resteasy/pom.xml

```diff
@@ -41,6 +41,11 @@
       <artifactId>beadledom-testing</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
You will want to add this dependency with the other test scoped dependencies. Dependencies are ordered compile first, then test.

Reply:
d376a7bc